examples/vite/vite.config.ts

```diff
@@ -0,0 +1,17 @@
+import {defineConfig} from 'vite';
+import {hydrogen, oxygen} from '@shopify/cli-hydrogen/experimental-vite';
+import {unstable_vitePlugin as remix} from '@remix-run/dev';
+import tsconfigPaths from 'vite-tsconfig-paths';
+
+export default defineConfig({
+  plugins: [
+    hydrogen(),
+    oxygen(),
+    remix({
+      buildDirectory: 'dist',
+      // TODO update Remix to fix this
+      publicPath: process.env.HYDROGEN_ASSET_BASE_URL,
+    }),
```

Comment:
With the template as is, won't a `remix.config.js` file get generated, even though it's not used by remix vite? Maybe that's fine for now while "vite" support for Hydrogen is unstable?